src/main/ruby/truffleruby/core/gc.rb

```diff
@@ -76,9 +76,15 @@ def garbage_collect
   end
 
   def self.stat(option = nil)
+    stats = Primitive.gc_stat()
```

Comment:
It would be more readable to use multiple assignment here, like:
```ruby
time, count, minor_count, major_count, committed, used = Primitive.gc_stat
```